cmd/server/cmd/server.go

```diff
@@ -308,6 +323,9 @@ func runServer(sa *serverArgs, tmplRepo template.Repository, printf, fatalf shar
 	grpcOptions = append(grpcOptions, grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(interceptors...)))
 
 	configManager.Register(adapterMgr)
+	if sa.useIL {
+		configManager.Register(ilEval)
```

Comment:
Why does config manager need eval? I think it would be nice to tease out these into their own layers as appropriate. the Evaluator interface seems to be highly overloaded.
(I won't take any action for this CL, but may look into it in the future).